sprints/dashboard/tasks.py

```diff
@@ -413,3 +434,146 @@ def unflag_tickets_task() -> None:
+    this should not cause any troubles.
+    """
+    with connect_to_jira() as conn:
+        session_name = get_next_poker_session_name(conn)
+        for cell in get_cells(conn):
+            if not settings.DEBUG:  # We really don't want to trigger this in the dev environment.
+                conn.create_poker_session(
+                    board_id=cell.board_id,
+                    name=session_name,
+                    issues=[],
+                    participants=[],
+                    scrum_masters=[conn.myself()['key']],
+                    send_invitations=False,
+                )
+
+
+@celery_app.task(ignore_result=True)
+def update_estimation_session_task() -> None:
+    """
+    Update estimation session's issues and participants.
+
+    If no issues exist for the session, then it will not be updated. The reasoning behind this has been described in the
+    `create_estimation_session_task` function.
+
+    This does not override the manual changes made to the session - i.e. if issue or user has been added manually to the
+    session, then it will be retained, as it merges available issues and participants with the applied ones.
+
+    FIXME: This adds all cell members as scrum masters to the session, because at the moment we do not have a way to
+           determine whether the member is a part of the core team. We can restrict this in the future, if needed.
+    """
+    with connect_to_jira() as conn:
+        session_name = get_next_poker_session_name(conn)
+        issues = get_unestimated_next_sprint_issues(conn)
+        # FIXME: This will not work for more than 1000 users. To support it, add `startAt` to handle the pagination.
+        all_users = conn.search_users("''", maxResults=1000)
+
+        for cell in get_cells(conn):
+            # FIXME: This will raise `IndexError` if a session does not exist. It can happen when this runs before the
+            #  `create_estimation_session_task`, e.g. when the sprint is created manually a moment before the full hour.
+            poker_session = conn.poker_sessions(cell.board_id, state="OPEN", name=session_name)[0]
+
+            # Get IDs of issues belonging only to a specific cell.
+            cell_issue_ids = set(issue.id for issue in issues if issue.key.startswith(cell.key))
+            current_issue_ids = set(conn.poker_session_results(poker_session.sessionId).keys())
+            all_issue_ids = list(current_issue_ids | cell_issue_ids)
+            if all_issue_ids:
+                # User's `name` and `key` are not always the same.
+                members = get_cell_member_names(conn, get_cell_members(conn.quickfilters(cell.board_id)))
+                member_keys = set(user.key for user in all_users if user.displayName in members)
+                current_member_keys = set(user.userKey for user in poker_session.participants)
+                all_member_keys = list(current_member_keys | member_keys)
```

Comment:
This means it doesn't make sense to remove newcomers after the Poker session has them, as they will be set again, right? Maybe add this to the docstring?

Reply:
Good idea, I added it to the docstring and the docs.